### ISSUE TITLE
LDDTool generates a 1C00 file when -V 1B00 is specified

### ISSUE THREAD

**Issue by jshughes** [NASA-PDS/pds4-information-model#322]
LDDTool generates a 1C00 file when -V 1B00 is specified on the command line. 

**Expected behavior**
LDDTool should generate a 1B00 file when -V 1B00 is specified on the command line. 

The problem was that the 1B00 directory contained a 1C00 config.properties file. The 1B00 config.properties files has been updated.

**Comment by rchenatjpl** [comment]
Am I using -V correctly? I don't see it doing anything. I'm not sure what should happen so attached are 4 variants of the same Ingest_LDD file. For each one, the output files are the same regardless of the presence of -V 1B00.
[Archive.zip](https://github.com/NASA-PDS/pds4-information-model/files/6595062/Archive.zip)

Last login: Thu Jun  3 17:06:07 on ttys006
% lddtool -pl im322lddinputBB.xml 

>>  INFO LDDTOOL Version: 13.1.0-SNAPSHOT
>>  INFO IM Version Id: 1.16.0.0
>>  INFO IM Namespace Id: pds
>>  INFO IM Label Version Id: 1.0
>>  INFO Date: Thu Jun 03 17:38:03 PDT 2021
>>  INFO PARENT_DIR: /Users/rchen/PDS4tools/lddtool
>>  INFO SCRIPT_DIR: /Users/rchen/PDS4tools/lddtool/bin
>>  INFO LIB_DIR: /Users/rchen/PDS4tools/lddtool/lib

>>> INFO Init:  - Config.Properties Namespace Id Using:pds
>>> INFO Init:  - LDD Dictionary_Type is Discipline
>>> WARNING Init:  - Config.Properties Namespace Id Not Found:geometry
>>> WARNING Header:  - New steward has been specified:geo
>>> WARNING Header:  - New namespace id has been specified:geometry

 -- LDDTool Execution Summary --
     3 warning(s)
     0 error(s)
     0 fatal error(s)

>>  INFO Exit(0)
% ls -tl | head
total 1768
-rw-rw-r--  1 rchen  703763885    2806 Jun  3 17:38 PDS4_GEOMETRY_1G00_1B00.xml
-rw-rw-r--  1 rchen  703763885    1173 Jun  3 17:38 PDS4_GEOMETRY_1G00_1B00.sch
-rw-rw-r--  1 rchen  703763885    3558 Jun  3 17:38 PDS4_GEOMETRY_1G00_1B00.xsd
-rw-rw-r--  1 rchen  703763885    1502 Jun  3 17:38 PDS4_GEOMETRY_1G00_1B00.csv
-rw-rw-r--  1 rchen  703763885    2850 Jun  3 17:38 PDS4_GEOMETRY_1G00_1B00.txt
-rw-r--r--@ 1 rchen  703763885    3216 Jun  3 17:36 im322lddinputGG.xml
-rw-r--r--  1 rchen  703763885    3216 Jun  3 17:36 im322lddinputGB.xml
-rw-r--r--  1 rchen  703763885    3216 Jun  3 17:35 im322lddinputBG.xml
-rw-r--r--@ 1 rchen  703763885    3216 Jun  3 17:33 im322lddinputBB.xml
% 
% 
% lddtool -V 1B00 -pl im322lddinputBB.xml

>>  INFO LDDTOOL Version: 13.1.0-SNAPSHOT
>>  INFO IM Version Id: 1.16.0.0
>>  INFO IM Namespace Id: pds
>>  INFO IM Label Version Id: 1.0
>>  INFO Date: Thu Jun 03 17:39:14 PDT 2021
>>  INFO PARENT_DIR: /Users/rchen/PDS4tools/lddtool
>>  INFO SCRIPT_DIR: /Users/rchen/PDS4tools/lddtool/bin
>>  INFO LIB_DIR: /Users/rchen/PDS4tools/lddtool/lib

>>> INFO Init:  - Config.Properties Namespace Id Using:pds
>>> INFO Init:  - LDD Dictionary_Type is Discipline
>>> WARNING Init:  - Config.Properties Namespace Id Not Found:geometry
>>> WARNING Header:  - New steward has been specified:geo
>>> WARNING Header:  - New namespace id has been specified:geometry

 -- LDDTool Execution Summary --
     3 warning(s)
     0 error(s)
     0 fatal error(s)

>>  INFO Exit(0)
% ls -tl | head
total 1768
-rw-rw-r--  1 rchen  703763885    2806 Jun  3 17:39 PDS4_GEOMETRY_1G00_1B00.xml
-rw-rw-r--  1 rchen  703763885    1173 Jun  3 17:39 PDS4_GEOMETRY_1G00_1B00.sch
-rw-rw-r--  1 rchen  703763885    3558 Jun  3 17:39 PDS4_GEOMETRY_1G00_1B00.xsd
-rw-rw-r--  1 rchen  703763885    1502 Jun  3 17:39 PDS4_GEOMETRY_1G00_1B00.csv
-rw-rw-r--  1 rchen  703763885    2850 Jun  3 17:39 PDS4_GEOMETRY_1G00_1B00.txt
-rw-r--r--@ 1 rchen  703763885    3216 Jun  3 17:36 im322lddinputGG.xml
-rw-r--r--  1 rchen  703763885    3216 Jun  3 17:36 im322lddinputGB.xml
-rw-r--r--  1 rchen  703763885    3216 Jun  3 17:35 im322lddinputBG.xml
-rw-r--r--@ 1 rchen  703763885    3216 Jun  3 17:33 im322lddinputBB.xml
% 
% 
% lddtool -pl im322lddinputGB.xml

>>  INFO LDDTOOL Version: 13.1.0-SNAPSHOT
>>  INFO IM Version Id: 1.16.0.0
>>  INFO IM Namespace Id: pds
>>  INFO IM Label Version Id: 1.0
>>  INFO Date: Thu Jun 03 17:39:41 PDT 2021
>>  INFO PARENT_DIR: /Users/rchen/PDS4tools/lddtool
>>  INFO SCRIPT_DIR: /Users/rchen/PDS4tools/lddtool/bin
>>  INFO LIB_DIR: /Users/rchen/PDS4tools/lddtool/lib

>>> INFO Init:  - Config.Properties Namespace Id Using:pds
>>> INFO Init:  - LDD Dictionary_Type is Discipline
>>> WARNING Init:  - Config.Properties Namespace Id Not Found:geometry
>>> WARNING Header:  - New steward has been specified:geo
>>> WARNING Header:  - New namespace id has been specified:geometry

 -- LDDTool Execution Summary --
     3 warning(s)
     0 error(s)
     0 fatal error(s)

>>  INFO Exit(0)
% ls -tl | head
total 1768
-rw-rw-r--  1 rchen  703763885    2806 Jun  3 17:39 PDS4_GEOMETRY_1G00_1B00.xml
-rw-rw-r--  1 rchen  703763885    1173 Jun  3 17:39 PDS4_GEOMETRY_1G00_1B00.sch
-rw-rw-r--  1 rchen  703763885    3558 Jun  3 17:39 PDS4_GEOMETRY_1G00_1B00.xsd
-rw-rw-r--  1 rchen  703763885    1502 Jun  3 17:39 PDS4_GEOMETRY_1G00_1B00.csv
-rw-rw-r--  1 rchen  703763885    2850 Jun  3 17:39 PDS4_GEOMETRY_1G00_1B00.txt
-rw-r--r--@ 1 rchen  703763885    3216 Jun  3 17:36 im322lddinputGG.xml
-rw-r--r--  1 rchen  703763885    3216 Jun  3 17:36 im322lddinputGB.xml
-rw-r--r--  1 rchen  703763885    3216 Jun  3 17:35 im322lddinputBG.xml
-rw-r--r--@ 1 rchen  703763885    3216 Jun  3 17:33 im322lddinputBB.xml
% 
% 
% 
% lddtool -V 1B00 -pl im322lddinputGB.xml

>>  INFO LDDTOOL Version: 13.1.0-SNAPSHOT
>>  INFO IM Version Id: 1.16.0.0
>>  INFO IM Namespace Id: pds
>>  INFO IM Label Version Id: 1.0
>>  INFO Date: Thu Jun 03 17:40:10 PDT 2021
>>  INFO PARENT_DIR: /Users/rchen/PDS4tools/lddtool
>>  INFO SCRIPT_DIR: /Users/rchen/PDS4tools/lddtool/bin
>>  INFO LIB_DIR: /Users/rchen/PDS4tools/lddtool/lib

>>> INFO Init:  - Config.Properties Namespace Id Using:pds
>>> INFO Init:  - LDD Dictionary_Type is Discipline
>>> WARNING Init:  - Config.Properties Namespace Id Not Found:geometry
>>> WARNING Header:  - New steward has been specified:geo
>>> WARNING Header:  - New namespace id has been specified:geometry

 -- LDDTool Execution Summary --
     3 warning(s)
     0 error(s)
     0 fatal error(s)

>>  INFO Exit(0)
% ls -tl | head
total 1768
-rw-rw-r--  1 rchen  703763885    2806 Jun  3 17:40 PDS4_GEOMETRY_1G00_1B00.xml
-rw-rw-r--  1 rchen  703763885    1173 Jun  3 17:40 PDS4_GEOMETRY_1G00_1B00.sch
-rw-rw-r--  1 rchen  703763885    3558 Jun  3 17:40 PDS4_GEOMETRY_1G00_1B00.xsd
-rw-rw-r--  1 rchen  703763885    1502 Jun  3 17:40 PDS4_GEOMETRY_1G00_1B00.csv
-rw-rw-r--  1 rchen  703763885    2850 Jun  3 17:40 PDS4_GEOMETRY_1G00_1B00.txt
-rw-r--r--@ 1 rchen  703763885    3216 Jun  3 17:36 im322lddinputGG.xml
-rw-r--r--  1 rchen  703763885    3216 Jun  3 17:36 im322lddinputGB.xml
-rw-r--r--  1 rchen  703763885    3216 Jun  3 17:35 im322lddinputBG.xml
-rw-r--r--@ 1 rchen  703763885    3216 Jun  3 17:33 im322lddinputBB.xml
% rm PDS4_GEOM*
% 
% 
% lddtool -pl im322lddinputBG.xml

>>  INFO LDDTOOL Version: 13.1.0-SNAPSHOT
>>  INFO IM Version Id: 1.16.0.0
>>  INFO IM Namespace Id: pds
>>  INFO IM Label Version Id: 1.0
>>  INFO Date: Thu Jun 03 17:40:43 PDT 2021
>>  INFO PARENT_DIR: /Users/rchen/PDS4tools/lddtool
>>  INFO SCRIPT_DIR: /Users/rchen/PDS4tools/lddtool/bin
>>  INFO LIB_DIR: /Users/rchen/PDS4tools/lddtool/lib

>>> INFO Init:  - Config.Properties Namespace Id Using:pds
>>> INFO Init:  - LDD Dictionary_Type is Discipline
>>> WARNING Init:  - Config.Properties Namespace Id Not Found:geometry
>>> WARNING Header:  - New steward has been specified:geo
>>> WARNING Header:  - New namespace id has been specified:geometry

 -- LDDTool Execution Summary --
     3 warning(s)
     0 error(s)
     0 fatal error(s)

>>  INFO Exit(0)
% ls -tl | head
total 1768
-rw-rw-r--  1 rchen  703763885    2806 Jun  3 17:40 PDS4_GEOMETRY_1G00_1G00.xml
-rw-rw-r--  1 rchen  703763885    1173 Jun  3 17:40 PDS4_GEOMETRY_1G00_1G00.sch
-rw-rw-r--  1 rchen  703763885    3558 Jun  3 17:40 PDS4_GEOMETRY_1G00_1G00.xsd
-rw-rw-r--  1 rchen  703763885    1502 Jun  3 17:40 PDS4_GEOMETRY_1G00_1G00.csv
-rw-rw-r--  1 rchen  703763885    2850 Jun  3 17:40 PDS4_GEOMETRY_1G00_1G00.txt
-rw-r--r--@ 1 rchen  703763885    3216 Jun  3 17:36 im322lddinputGG.xml
-rw-r--r--  1 rchen  703763885    3216 Jun  3 17:36 im322lddinputGB.xml
-rw-r--r--  1 rchen  703763885    3216 Jun  3 17:35 im322lddinputBG.xml
-rw-r--r--@ 1 rchen  703763885    3216 Jun  3 17:33 im322lddinputBB.xml
% 
% 
% 
% lddtool -V 1B00 -pl im322lddinputBG.xml

>>  INFO LDDTOOL Version: 13.1.0-SNAPSHOT
>>  INFO IM Version Id: 1.16.0.0
>>  INFO IM Namespace Id: pds
>>  INFO IM Label Version Id: 1.0
>>  INFO Date: Thu Jun 03 17:41:00 PDT 2021
>>  INFO PARENT_DIR: /Users/rchen/PDS4tools/lddtool
>>  INFO SCRIPT_DIR: /Users/rchen/PDS4tools/lddtool/bin
>>  INFO LIB_DIR: /Users/rchen/PDS4tools/lddtool/lib

>>> INFO Init:  - Config.Properties Namespace Id Using:pds
>>> INFO Init:  - LDD Dictionary_Type is Discipline
>>> WARNING Init:  - Config.Properties Namespace Id Not Found:geometry
>>> WARNING Header:  - New steward has been specified:geo
>>> WARNING Header:  - New namespace id has been specified:geometry

 -- LDDTool Execution Summary --
     3 warning(s)
     0 error(s)
     0 fatal error(s)

>>  INFO Exit(0)
% ls -tl | head
total 1768
-rw-rw-r--  1 rchen  703763885    2806 Jun  3 17:41 PDS4_GEOMETRY_1G00_1G00.xml
-rw-rw-r--  1 rchen  703763885    1173 Jun  3 17:41 PDS4_GEOMETRY_1G00_1G00.sch
-rw-rw-r--  1 rchen  703763885    3558 Jun  3 17:41 PDS4_GEOMETRY_1G00_1G00.xsd
-rw-rw-r--  1 rchen  703763885    1502 Jun  3 17:41 PDS4_GEOMETRY_1G00_1G00.csv
-rw-rw-r--  1 rchen  703763885    2850 Jun  3 17:41 PDS4_GEOMETRY_1G00_1G00.txt
-rw-r--r--@ 1 rchen  703763885    3216 Jun  3 17:36 im322lddinputGG.xml
-rw-r--r--  1 rchen  703763885    3216 Jun  3 17:36 im322lddinputGB.xml
-rw-r--r--  1 rchen  703763885    3216 Jun  3 17:35 im322lddinputBG.xml
-rw-r--r--@ 1 rchen  703763885    3216 Jun  3 17:33 im322lddinputBB.xml
% 
% 
% lddtool -pl im322lddinputGG.xml

>>  INFO LDDTOOL Version: 13.1.0-SNAPSHOT
>>  INFO IM Version Id: 1.16.0.0
>>  INFO IM Namespace Id: pds
>>  INFO IM Label Version Id: 1.0
>>  INFO Date: Thu Jun 03 17:41:20 PDT 2021
>>  INFO PARENT_DIR: /Users/rchen/PDS4tools/lddtool
>>  INFO SCRIPT_DIR: /Users/rchen/PDS4tools/lddtool/bin
>>  INFO LIB_DIR: /Users/rchen/PDS4tools/lddtool/lib

>>> INFO Init:  - Config.Properties Namespace Id Using:pds
>>> INFO Init:  - LDD Dictionary_Type is Discipline
>>> WARNING Init:  - Config.Properties Namespace Id Not Found:geometry
>>> WARNING Header:  - New steward has been specified:geo
>>> WARNING Header:  - New namespace id has been specified:geometry

 -- LDDTool Execution Summary --
     3 warning(s)
     0 error(s)
     0 fatal error(s)

>>  INFO Exit(0)
% ls -tl | head
total 1768
-rw-rw-r--  1 rchen  703763885    2806 Jun  3 17:41 PDS4_GEOMETRY_1G00_1G00.xml
-rw-rw-r--  1 rchen  703763885    1173 Jun  3 17:41 PDS4_GEOMETRY_1G00_1G00.sch
-rw-rw-r--  1 rchen  703763885    3558 Jun  3 17:41 PDS4_GEOMETRY_1G00_1G00.xsd
-rw-rw-r--  1 rchen  703763885    1502 Jun  3 17:41 PDS4_GEOMETRY_1G00_1G00.csv
-rw-rw-r--  1 rchen  703763885    2850 Jun  3 17:41 PDS4_GEOMETRY_1G00_1G00.txt
-rw-r--r--@ 1 rchen  703763885    3216 Jun  3 17:36 im322lddinputGG.xml
-rw-r--r--  1 rchen  703763885    3216 Jun  3 17:36 im322lddinputGB.xml
-rw-r--r--  1 rchen  703763885    3216 Jun  3 17:35 im322lddinputBG.xml
-rw-r--r--@ 1 rchen  703763885    3216 Jun  3 17:33 im322lddinputBB.xml
% 
% 
% lddtool -V 1B00 -pl im322lddinputGG.xml

>>  INFO LDDTOOL Version: 13.1.0-SNAPSHOT
>>  INFO IM Version Id: 1.16.0.0
>>  INFO IM Namespace Id: pds
>>  INFO IM Label Version Id: 1.0
>>  INFO Date: Thu Jun 03 17:41:33 PDT 2021
>>  INFO PARENT_DIR: /Users/rchen/PDS4tools/lddtool
>>  INFO SCRIPT_DIR: /Users/rchen/PDS4tools/lddtool/bin
>>  INFO LIB_DIR: /Users/rchen/PDS4tools/lddtool/lib

>>> INFO Init:  - Config.Properties Namespace Id Using:pds
>>> INFO Init:  - LDD Dictionary_Type is Discipline
>>> WARNING Init:  - Config.Properties Namespace Id Not Found:geometry
>>> WARNING Header:  - New steward has been specified:geo
>>> WARNING Header:  - New namespace id has been specified:geometry

 -- LDDTool Execution Summary --
     3 warning(s)
     0 error(s)
     0 fatal error(s)

>>  INFO Exit(0)
% ls -tl | head
total 1768
-rw-rw-r--  1 rchen  703763885    2806 Jun  3 17:41 PDS4_GEOMETRY_1G00_1G00.xml
-rw-rw-r--  1 rchen  703763885    1173 Jun  3 17:41 PDS4_GEOMETRY_1G00_1G00.sch
-rw-rw-r--  1 rchen  703763885    3558 Jun  3 17:41 PDS4_GEOMETRY_1G00_1G00.xsd
-rw-rw-r--  1 rchen  703763885    1502 Jun  3 17:41 PDS4_GEOMETRY_1G00_1G00.csv
-rw-rw-r--  1 rchen  703763885    2850 Jun  3 17:41 PDS4_GEOMETRY_1G00_1G00.txt
-rw-r--r--@ 1 rchen  703763885    3216 Jun  3 17:36 im322lddinputGG.xml
-rw-r--r--  1 rchen  703763885    3216 Jun  3 17:36 im322lddinputGB.xml
-rw-r--r--  1 rchen  703763885    3216 Jun  3 17:35 im322lddinputBG.xml
-rw-r--r--@ 1 rchen  703763885    3216 Jun  3 17:33 im322lddinputBB.xml

**Comment by jordanpadams** [comment]
@rchenatjpl for the moment, ordering of the arguments matters. see below.

```
$ ~/Downloads/lddtool-13.0.2/bin/lddtool -lp PDS4_IMG_IngestLDD.xml -V 1B00
$ ls
-rw-rw-r--   1 jpadams  staff   217K Jun  3 19:39 PDS4_IMG_1B00_1810.csv
-rw-rw-r--   1 jpadams  staff    38K Jun  3 19:39 PDS4_IMG_1B00_1810.sch
-rw-rw-r--   1 jpadams  staff   164K Jun  3 19:39 PDS4_IMG_1B00_1810.txt
-rw-rw-r--   1 jpadams  staff   2.7K Jun  3 19:39 PDS4_IMG_1B00_1810.xml
-rw-rw-r--   1 jpadams  staff   203K Jun  3 19:39 PDS4_IMG_1B00_1810.xsd
```

OR 

```
$ ~/Downloads/lddtool-13.0.2/bin/lddtool -lp -V 1B00 PDS4_IMG_IngestLDD.xml
$ ls
-rw-rw-r--   1 jpadams  staff   217K Jun  3 19:39 PDS4_IMG_1B00_1810.csv
-rw-rw-r--   1 jpadams  staff    38K Jun  3 19:39 PDS4_IMG_1B00_1810.sch
-rw-rw-r--   1 jpadams  staff   164K Jun  3 19:39 PDS4_IMG_1B00_1810.txt
-rw-rw-r--   1 jpadams  staff   2.7K Jun  3 19:39 PDS4_IMG_1B00_1810.xml
-rw-rw-r--   1 jpadams  staff   203K Jun  3 19:39 PDS4_IMG_1B00_1810.xsd
```

this will be fixed with next build:

**Comment by jordanpadams** [comment]
will be fixed by #240
